src/main/resources/application.intg.conf

```diff
@@ -0,0 +1,15 @@
+include "application.conf"
+
+gov_uk_notify {
+  endpoint = "https://api.notifications.service.gov.uk"
+  api_key = "arbitrary_placeholder"
+  api_key = ${?GOV_UK_NOTIFY_API_KEY}
+  on = false
+  on = ${?SEND_GOV_UK_NOTIFICATIONS}
+  transfer_complete_template_id = "arbitrary_placeholder"
+  transfer_complete_template_id = "6fe48fcd-24bf-4a17-b4fe-d121b12a1b4d"
+  metadata_review_template_id = "arbitrary_placeholder"
+  metadata_review_template_id = ${?METADATA_REVIEW_TEMPLATE_ID}
```

Comment:
Are you going to add the template ids for the other two templates?

Reply:
Oops forgot about that. Will add them. There will be an additional 2 templates added but I will do that in my other PR related to sending the emails. So I've just added some default ones for now.